manage.py

```diff
@@ -1,9 +1,45 @@
-#!/usr/bin/env python
 """Django's command-line utility for administrative tasks."""
-import os
 import sys
+import os
+from contextlib import nullcontext, contextmanager
+
+
+@contextmanager
+def coverage_context():
+    """Context manager to collect code coverage data.
+
+    Uses coverage package to collect coverage, generate reports
+    and print coverage percentage.
+    """
+    # pylint: disable-next=import-outside-toplevel
+    from coverage import Coverage
```

Comment:
It's possible to update `.pylintrc` instead of making the inline pylint-disable comment:
```ini
allow-any-import-level =
  coverage,
```
But it's up to you, keeping it here is also fine.

Reply:
Ok, I will change it. Thanks.

Reply:
Marking this as resolved.